### PR TITLE
proc: change 'step' command so that it steps through go statements

### PR DESCRIPTION
**Pull request by aarzilli**
Change 'step' command so that when stepping into a 'go statement' the
debugger will stop on the newly created goroutine, instead of just
stepping over the go statement.

### CODE REVIEW

File: pkg/proc/target_exec.go [hunk 1]
```diff
@@ -999,6 +1010,107 @@ func setStepIntoBreakpoint(dbp *Target, curfn *Function, text []AsmInstruction,
...
+			callfile = instr.Loc.File
+			callline = instr.Loc.Line
+			break
+		}
+	}
+	if callfile == "" {
+		logflags.DebuggerLogger().Error("could not find " + runtimeRunqput + " call in " + runtimeNewprocFunc1)
+		return
+	}
+	var pcs []uint64
```
Comment (derekparker) [inline pkg/proc/target_exec.go:1045]: We only ever use `pcs[0]` after this, so why not just a `var pc uint64` and assign the first address where `pcstmt.Stmt == true`?

Comment (aarzilli) [reply pkg/proc/target_exec.go]: Done.

**Review by derekparker — APPROVED** [review]
LGTM

**Comment by derekparker** [comment]
@aarzilli needs rebase.

**Comment by aarzilli** [comment]
Done

**Pull request merged**